charts/connectors/deployment.go

```diff
@@ -182,6 +181,12 @@ func Deployment(dot *helmette.Dot) *appsv1.Deployment {
 			},
 		},
 	}
+
+	if !helmette.Empty(values.Deployment.Command) {
```

Comment:
This fix doesn't make sense to me.

```go
	Command []string `json:"command,omitempty" protobuf:"bytes,3,rep,name=command"`
```

From the golang (Kubernetes) perspective, not setting `Command` and explicitly setting `Command` to nil are equivalent. Is there something I'm overlooking?

Reply:
transpiled template code is always setting command regardless of the command.

Go code will work. Template not. You can see the diff in the golden file.